test/UID2.Client.Test/EncryptionTestsV4.cs

```diff
@@ -12,12 +12,13 @@ public class EncryptionTestsV4
         const long SITE_KEY_ID = 165;
         const int SITE_ID = 9000; //const required for switch expression
         const int SITE_ID2 = 2;
+        const string EXAMPLE_EMAIL_UID = "ywsvDNINiZOVSsfkHpLpSJzXzhr6Jx9Z/4Q0+lsEUvM=";
+        const string EXAMPLE_PHONE_UID = "BFOsW2SkK0egqbfyiALtpti5G/cG+PcEvjkoHl56rEV8";
```

Comment:
worth commenting EXAMPLE_EMAIL_UID is v2 raw UID and EXAMPLE_EMAIL_UID is v3?

Reply:
or rather than a comment, use EXAMPLE_EMAIL_RAW_UID2_V2 etc